code/modules/power/multiz.dm

```diff
@@ -1,19 +1,38 @@
-/obj/structure/cable/multilayer/multiz //This bridges powernets betwen Z levels
+/obj/structure/cable/multiz //This bridges powernets betwen Z levels
 	name = "multi z layer cable hub"
 	desc = "A flexible, superconducting insulated multi Z layer hub for heavy-duty multi Z power transfer."
 	icon = 'icons/obj/power.dmi'
 	icon_state = "cablerelay-on"
```

Comment:
`cable_bridge` may be a better state, as since this (seems) to act as a cardinally connected omnicable, the sprite would imply otherwise. Ideally, if you could figure out a way to make normal cables handle this, maybe treating `UP` and `DOWN` as valid cable directions, that'd be great, as I'd like to obsolete magic equipment like this entirely.